pkg/pillar/vault/handler_zfs.go

```diff
@@ -398,29 +406,16 @@ func CreateZvolVault(log *base.LogObject, datasetName string, zfsKeyFile string,
 }
 
 // CreateZvolEtcd Create and mount an empty vault dataset zvol
-func CreateZvolEtcd(log *base.LogObject, datasetName string) error {
-	// Remaining space in the pool
-	sizeBytes := uint64(0)
-	etcdSizeBytes := uint64(1024 * 1024 * 1024 * 1)
-
-	parentDatasetName := datasetName
-	if strings.Contains(parentDatasetName, "/") {
-		datasetParts := strings.Split(parentDatasetName, "/")
-		parentDatasetName = datasetParts[0]
-	}
-
-	sizeBytes, err := zfs.GetDatasetAvailableBytes(parentDatasetName)
+func CreateZvolEtcd(log *base.LogObject, datasetName string, zfsKeyFile string, encrypted bool) error {
+	etcdSizeGb, err := getEtcdSizeSetting()
 	if err != nil {
-		return fmt.Errorf("Dataset %s available bytes read error: %v", parentDatasetName, err)
-	}
-
-	if sizeBytes > (1024 * 1024 * 1024 * 10) {
-		etcdSizeBytes = 1024 * 1024 * 1024 * 10
+		log.Errorf("Using defaults, can't read etcd size setting: %v", err)
```

Comment:
Why not print the resulting default value which is used (etcSizeGb)?

Reply:
Done